src/configuration/services.md

```diff
@@ -70,6 +70,14 @@ For example, the current default storage amount per project is 5GB (meaning 5120
 
 In order for a service to be available to an application in your project (Platform.sh supports not only multiple backends but also multiple applications in each project) you will need to refer to it in the [.platform.app.yaml](/configuration/app-containers.md) file which configures the *relationships* between applications and services.
 
+
+## Endpoints
+
+All services offer one or more `endpoints`.  An endpoint is simply a named set of credentials that can be used to access the service from other applications or services in your project.  Only some services support multiple user-defined endpoints.  If you do not specify one then one will be created with a standard defined name, generally the name of the service type (e.g., `mysql` or `solr`).  An application container, defined by a `.platform.app.yaml` file, always exposes and endpoint named `http` to allow the [router](/configuration/routes.md) to forward requests to it.
```

Comment:
If is _is_ a set of credentials, rather than just something that 1-1 maps to a set of credentials, I'd prefer we call it a "user", or... a "set of credentials"